Src/AutoFixture.NUnit3.UnitTest/AutoDataAttributeTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class AutoDataAttributeTest
+    {
+        [Test]
+        public void ExtendsNUnitAttribute()
+        {
+            Assert.That(new AutoDataAttribute(), Is.InstanceOf<NUnitAttribute>());
+        }
+
+        [Test]
+        public void ImplementsITestBuilderITestCaseDataAndIImplyFixture()
+        {
+            var autoDataFixture = new AutoDataAttribute();
+            Assert.That(autoDataFixture, Is.InstanceOf<ITestBuilder>());
+            Assert.That(autoDataFixture, Is.InstanceOf<ITestCaseData>());
+            Assert.That(autoDataFixture, Is.InstanceOf<IImplyFixture>());
```

Comment:
Why is it important that the class implements `IImplyFixture`?

I tried removing the interface implementation from `AutoDataAttribute`, and commented out this line here, and the build still succeeds.